### DISCUSSION TITLE
New release/release cycle

### DISCUSSION THREAD

**Discussion by callumrollo** [GliderToolsCommunity/GliderTools#122] [Q&A] [answered]
General question: When is a new release on PyPI/conda-forge made and how is this timing decided?

Specific request: Can we get a new release this week please? Could use the latest and greatest version for our demo next week, binder needs the most recent GliderTools on conda-forge to work. Ideally we'd get astral #121 merged first

Thanks!

**Comment by jbusecke** [comment, marked as answer]
FYI you can always install the latest version by adding 
```yml
- ...
- pip:
    - git+https://github.com/GliderToolsCommunity/GliderTools.git
```
to your environment file (note that this only works with the https link to the repo, not the ssh one).

For the more general release schedule, I think releasing often is a good practice (that I often forget myself). All of the Pypi/conda stuff should be automated, so all that is required is a release on github.

**Reply by jbusecke** [reply]
Happy to make a release now or wait on #121.

**Reply by callumrollo** [reply]
Thanks for the explanation, a release after #121 would be great
